lib/action_view/component/conversion.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ActionView
+  module Component # :nodoc:
```

Comment:
```suggestion
  module ActiveModel # :nodoc:
```
And previous line should be removed. This is a monkey patch from #129

Reply:
It's already there on the [last line](https://github.com/github/actionview-component/pull/138/files#diff-7348f38d58d7fe263398878a3236faa2R21) of `component.rb` file!

Reply:
Ahh got it! Nice one